### PR TITLE
What's new 11.7.3.0

### PR DESCRIPTION
**Pull request by krzychu124**
_New_
- Improved bus/trolleybus stop path #1688, #1690

_Fixed_
- Cargo truck pathfinding via `CargoVehicle` networks (cargo stations/terminals) #1701, #1703
- Lane connector on overlapping lanes #1706
- Parking signs on lanes with bus stop #1707

_Updated_
- Minor changes in debug tools #1696, #1697
- TMPE API Updates compatibility with other mods #1689 #1692

### CODE REVIEW

File: TLM/TLM/Resources/whats_new.txt [hunk 1]
```diff
@@ -1,10 +1,12 @@
...
+[Released] Dec 13th 2022
+[Link] tmpe-v11730-stable-13122022
 [Stable]
-[Meta] Compatibility patch for the game update 1.15.1-f4 (krzychu124)
-[New] Clear all TM:PE rules from selected node #692, #1668 (kianzarrin)
-[Fixed] Timed Traffic Lights wait = 0 issues #1497, #1674 (kianzarrin)
-[Updated] Minor changes to Timed Traffic Lights sensitivity slider #1670 (kianzarrin)
-[Updated] Pedestrian zone roads have now lower priority when setting up high priority road #1653 (kianzarrin)
+[Meta] Compatibility patch for the game update 1.16.0-f3 (krzychu124)
+[New] Improved bus/trolleybus stop path #1688, #1690 (kianzarrin)
```
Comment (kianzarrin) [inline TLM/TLM/Resources/whats_new.txt:6]: if this is visible in stable whats new panel then maybe rephrase it in a less abstract way:
buses pull to side faster at bus stops.

Comment (krzychu124) [reply TLM/TLM/Resources/whats_new.txt:6]: maybe not faster but earlier?

**Review by kvakvs — APPROVED** [review]
